tests/foreman/cli/test_activationkey.py

```diff
@@ -1307,7 +1307,7 @@ def test_update_autoattach_2(self):
                 self.assertEqual(
                     u'Activation key updated', result[0]['message'])
 
-    def test_negative_update_auto_attach(self):
+    def test_invalid_update_auto_attach(self):
```

Comment:
particular reason for that? It is not the right idea to change 'negative' on 'invalid' as it will be hard to run 'test_negative*' template in future from command line